zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/PublishNotification.java

```diff
@@ -52,27 +49,39 @@ public PublishNotification(Collection collection) {
     }
 
     public void sendNotification(EventType eventType) {
-        logInfo("Sending publish notification to website").addParameter("eventType", eventType.name()).log();
+        Host host = null;
         try (Http http = new Http()) {
-            for (Host host : websiteHosts) {
+            for (Host h : websiteHosts) {
+                host = h;
+                logInfo("sending publish notification to website host")
+                        .collectionId(payload.collectionId)
+                        .addParameter("websiteHost", host.toString())
```

Comment:
Other go projects are using snake casing, but I guess if Zebedee is already using camel casing it may be worth sticking with that